backend/src/main/java/com/crm/api/v1/CarController.java

```diff
@@ -0,0 +1,26 @@
+package com.crm.api.v1;
+
+import com.crm.dto.response.CarResponse;
+import com.crm.service.CarServiceImpl;
+import org.springframework.data.domain.Page;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/cars")
```

Comment:
Should be only /cars. Please configure basePath: /api

Reply:
Done